cfg/export.ads

```diff
@@ -0,0 +1,8 @@
+artifacts builderVersion: "1.1", {
+
+  group "com.sap.bds.ats-altiscale", {
+    artifact "pickling", {
+      file "$gendir/src/picklingrpmbuild/alti-pickling-0.11.x.rpm"
```

Comment:
This should be alti-picking-${buildVersion} Kavish. And it shouldn't be x. It should be a number

Reply:
Branch is defined as 0.11.x

Reply:
I think we should still use ${buildVersion} and not hard code it - it seems to be picking up the version from the VERSION file

Reply:
Oh and I cannot add a comment below Kavish but we shouldn't check in import folder which is created during the build process....

Reply:
agree. The file https://github.com/Altiscale/pickling/blob/pickling-xmake-dev/cfg/VERSION includes the `0.11.x` so we should be using that instead of a hardcoded string here.

Reply:
I'm okay with keeping the hardcoded version since we won't be updating this branch

Reply:
Keeping the hardcoded version as is. 
${buildVersion} appends SNAPSHOT
and for other variables, it gives 'variable not found'

Reply:
That's right. These are snapshot builds. When you build an OD build it becomes a release build. So buildVersion is doing the right thing